### ISSUE TITLE
"The App Name you entered has already been used" error on spaceship, but not through iTunes UI

### ISSUE THREAD

**Issue by Korcholis** [fastlane/fastlane#6314]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)

### Issue Description

I'm using `spaceship` to update  many iTunes entries at the same time, and after the new name length limitation, I had to change the titles. Sadly I'm getting the error _"The App Name you entered has already been used"_. I already read many issues (#1545 and #2121), and that they solve it by changing the name in the textfiles. I do it by code (I need to generate them on the fly), so I modify the code to change the `app_details.name[localised]`, but no chance.

The weird thing comes when I try to change it from the iTunes UI itself. It **works** from there. I write the same titles in each locale I generate in the app and it doesn't complain about anything.

A simplified piece of the code that runs the process is:

```
Spaceship::Tunes.login(Config::LOGIN[:email],Config::LOGIN[:pwd])
app = Spaceship::Tunes::Application.find(:app_number)
details = app.details
locale = "en-US" # for instance
details.name[locale] = generated_title
details.save! # It crashes here
```

### Environment

fastlane version (run `fastlane -v`): fastlane 1.104.0

Do you use bundler to execute fastlane (i.e. `bundle exec fastlane`)? No

Do you use a Ruby environment manager (e.g. `chruby`, `rbenv`, `rvm`)? No

**Comment by KrauseFx** [comment]
Do you have any other languages enabled? Can you post the complete tree of your `./fastlane` directory?

**Comment by Korcholis** [comment]
Yeah, I have all the languages available in the iTunes api (all but Indonesian). And well, as I mentioned, I'm doing all the processing by hand using `spaceship`, so basically no `./fastlane`.

**I managed to solve it, kind of**. It looks like I had to switch my default language from the `en-US` to another one. Then, I was able to put the names as I wanted, and afterwards I could move back to `en-US` as the primary language. Pretty weird, but in accordance with other people's troubles.

This still makes no sense to me. I assume it's an API problem and there's nothing in your hand to sort this out, but the iTunes interface itself worked without major troubles, so there's something smelly in their API they may not disclose.
